src/app/utils/encrypt-submission.ts

```diff
@@ -0,0 +1,56 @@
+import { Request } from 'express'
+
+import { IPopulatedForm } from 'src/types'
+
+import { createLoggerWithLabel } from '../../config/logger'
+import { getProcessedResponses } from '../modules/submission/submission.service'
+
+import { checkIsEncryptedEncoding } from './encryption'
+import { createReqMeta } from './request'
+
+const logger = createLoggerWithLabel(module)
+
+type ReqWithForm<T> = T & { form: IPopulatedForm }
+
+export const isValidEncryptSubmission = (req: Request): boolean => {
```

Comment:
I don't think we need to create the `isValidEncryptSubmission` function, can't we use the `checkIsEncryptedEncoding` function directly?

I think we can update the `checkIsEncryptedEncoding` function into a neverthrown function instead.